index.html

```diff
@@ -0,0 +1,96 @@
+    
+    <form action="" id="survey-form">
+      <div class="mb-3">
+      <label for="name" id="name-label">Name:</label>
+      <input class="form-control" type="text" name="name" id="name" placeholder="Put your name here!" required>
+      </div>
+
+      <div class="mb-3">
+      <label for="email" id="email-label">Email:</label>
+      <input class="form-control" type="email" name="email" id="email" placeholder="Put your email here!" required>
+      </div>
+
+      <div class="mb-3">
+      <label for="number" id="number-label">Number</label>
+      <input class="form-control" type="number" name="number" id="number" min="0" max="100" placeholder="Put a number between 0 and 100!">
+      </div>
+
+      <div class="mb-3">
+      <label for="dropdown" id="dropdown-label">Which option best describes your current role?</label>
+      <select class="form-select" name="dropdown" id="dropdown">
+        <option value="Student">Student</option>
+        <option value="Full_time">Full time job</option>
+      </select>
+      </div>
+      <label for="radio1" id="radio1-label">Select an option</label>
+      <div class="mb-3 form-check">
+      
+      <input class="form-check-input" type="radio" name="radio1" id="radio1" value="value1">value1
+      </div>
+      <div class="mb-3 form-check">
+      <input class="form-check-input" type="radio" name="radio1" id="radio2" value="value2">value2
+      </div>
+      
+
+      <div class="mb-3 form-check">
+      <input class="form-check-input" type="checkbox" name="transport_type1" id="transport_type1" value="Car">
+     
+      <label for="transport_type1" id="transport1-label">Car</label>
+    </div>
+      <div class="mb-3 form-check">
+      <input class="form-check-input" type="checkbox" name="transport_type2" id="transport_type2" value="Plane">
+      
+      <label for="transport_type2" id="transport2-label">Plane</label>
+    </div>
+      <div class="mb-3 form-check">
+    
+      <input class="form-check-input" type="checkbox" name="transport_type3" id="transport_type3" value="Walk">
+      <label for="transport_type3" id="transport3-label">Walk</label>
+    </div>
+      <div class="mb-3 form-check">
+      <input class="form-check-input" type="checkbox" name="transport_type4" id="transport_type4" value="Bike">
+      <label for="transport_type4" id="transport4-label">Bike</label>
+      </div>
+
+      <div class="mb-3">
+      <label for="comments" id="comments-label">Comments:</label>
+      <textarea class="form-control" name="comments" id="comments" cols="30" rows="10" placeholder="Put your comments here!"></textarea>
+      </div>
+
+      <div class="mb-3">
+      <input class="btn btn-primary mb-3" type="submit" id="submit">
+      </div>
+    </form>
+    </div>
+
+
+
+<!--  
+  </div>
+-->   
```

Comment:
:pray:  Please remember to delete all comments from your codebase. This is to get a clean code.